apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/WalletAccounts.tsx

```diff
@@ -121,6 +154,24 @@ export const WalletAccounts = ({ isPopup, onBack }: { isPopup: boolean; onBack:
           />
         </div>
       </div>
+      {enableAccountPasswordPrompt.isOpen && (
```

Comment:
Just a matter of keeping it consistent with the other dialogs
```suggestion
      {
      <EnableAccountPasswordPrompt
           open={enableAccountPasswordPrompt.isOpen}

```

Reply:
@lucas-barros i will add a comment: there is actually a big difference between these two ways of doing it. The way you see here will ensure that the password in the dialog will not be "stored" while the accounts menu stays open (because the component is only rendered conditionally vs. passing a prop) … for the other drawers its not important

Reply:
Makes sense, thanks for clarifying.